Practical Example/RW_interview_prep2/RW_interview_prep/Extension/Extensions.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Extensions.swift
+//  RW-takeHomeProject
+//
+//  Created by Duc Dang on 8/25/20.
+//  Copyright © 2020 Duc Dang. All rights reserved.
+//
+
+import Foundation
+
+extension Date {
+   
+   func convertToMonthDayYearFormat() -> String {
+       let formatter = DateFormatter()
+       formatter.dateFormat = "MMM dd yyyy"
+       
+       return formatter.string(from: self)
```

Comment:
If you're formatting dates for display to the user, it's a good idea to make sure the date is localised to what they expect.

```suggestion
let formatter = DateFormatter()
formatter.locale = .current
formatter.dateStyle = .medium
formatter.timeStyle = .none

return formatter.string(from: self)
```

This way, the order of the day, month and year will be what the user expects and is used to.